docs/arma3/getstarted/a3sync/README.md

```diff
@@ -0,0 +1,121 @@
+# Using Arma3Sync
+
+## What
+
+Arma3Sync is a powerful third-party launcher for Arma 3. It supports the use of groups of mods - `modsets` - modsets can be defined by individual users or an event host, and they can be loaded independently of each other.
+
+Arma3Sync also simplifies downloading mods from many different sources - known as `repositories`.
+
+Anrop's Arma3Sync repository is integrated with the operation pages on Anrop.se.  
+This enables users to browse either the full repository containing all mods currently being hosted in the repository, or event-specific subrepositories containing only mods that are part of a specific operation's modset.
+
+## Why
+
+For users who want better control of when mods are downloaded or updated, and where they are downloaded to and loaded into the game from.
+
+A number of additional options are available, but do not differ much from the basic Arma 3 launcher for normal users.
+
+## Installation
+
+### Step 1
+
+[Download Arma3Sync by clicking this link.](http://hosted.anrop.se/arma3sync.exe) run the installer and then start Arma3Sync when the installer finishes.
+
+You will then be presented with the following screen, asking you for the path to your Arma 3 installation folder.
+
+Find your Steam installation folder, from there the path to the Arma 3 folder is:  
+`\Steam\steamapps\common\Arma 3\`
+
+![Select Arma 3 dir](./assets/select_arma3_dir.png)
+
+### Step 2
+
+Before doing anything else we need to decide to where the mods shall be downloaded.
+
+In Windows Explorer navigate to your Arma 3 installation folder.  
+Create a folder called `armamods` there.   
```

Comment:
Shouldn't this be Anrop?

Reply:
Sure, gotta take a new picture then but that's easily resolved.